### PR TITLE
#706 tinyMCE to approve modal

### PR DESCRIPTION
**Pull request by Satendra-SR**
#706 

### Changes include

- Added tinyMCE (richtext editor) to approve modal.

### CODE REVIEW

File: resources/views/hr/application/approve-applicant-modal.blade.php [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="modal fade" id="approve_application" tabindex="-1" role="dialog" aria-labelledby="approve_application" aria-hidden="true" v-if="selectedAction == 'approve'">
```
Comment (abhishek-pokhriyal) [inline resources/views/hr/application/approve-applicant-modal.blade.php]: why is this removed? what was its purpose originally?

Comment (Satendra-SR) [reply resources/views/hr/application/approve-applicant-modal.blade.php]: @abhishek-pokhriyal, tinyMCE was not working with `Vue-js v-if` and when I removed it, the modal working as expected.
